tests/parser/ast_utils/test_ast_dict.py

```diff
@@ -52,7 +52,7 @@ def test_basic_ast():
             "node_id": 4,
             "src": "4:6:0",
         },
-        "ast_type": "VariableDef",
+        "ast_type": "VariableDecl",
```

Comment:
Can it be spelt out like `VariableDeclaration`?

Reply:
Hmm -- I think having a shorter AST name here is good as it's in line with other abbreviations in the AST, e.x. FunctionDef, AnnAssign, etc. Plus, `Decl` as a shortening is industry standard, see for ex. golang and rust grammars

https://go.dev/ref/spec#Declarations_and_scope
https://github.com/jorendorff/rust-grammar/blob/372f1aa092a298d789c620ee0e0717d8166a4829/Rust.g4#L36